dev-requirements.txt

```diff
@@ -5,7 +5,7 @@ pytest-xdist==1.34.0
 #pytest-mypy
 coverage
 selenium
-sphinx
+sphinx==3.2.1
```

Comment:
Is pinning this required?

Reply:
Building the docs fails with newer versions of sphinx, but that should be a different PR to this. I will remove this change in this PR.

Reply:
Sounds good, thanks!